rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/terminal/TerminalIDEServer.java

```diff
@@ -224,7 +226,7 @@ public void warning(WarningMessage param) {
 
     @Override
     public void registerLocations(RegisterLocationsParameters param) {
-        new BasicIDEServices(null).registerLocations(param.getScheme(), param.getAuthority(), param.getMapping());
+        new BasicIDEServices(new PrintWriter(PrintWriter.nullWriter()), new NullRascalMonitor()).registerLocations(param.getScheme(), param.getAuthority(), param.getMapping());
```

Comment:
Perhaps we can just inline the default method here, because it skips all the monitor variables anyway:

```
default void registerLocations(IString scheme, IString auth, IMap map) {
	  URIResolverRegistry.getInstance().registerLogical(new LogicalMapResolver(scheme.getValue(), auth.getValue(), map));
	}

  default void unregisterLocations(IString scheme, IString auth) {
    URIResolverRegistry.getInstance().unregisterLogical(scheme.getValue(), auth.getValue());
	}
```

Reply:
that would cause a clone of that code, I would rather not.

Reply:
ok, refactored that code a bit.